### DISCUSSION TITLE
Lubuntu não está imprimindo documentos de intervalo específico.

### DISCUSSION THREAD

**Discussion by Gilvan525** [lxqt/lxqt#2484] [Ideas]
Há semanas que eu uso o S.O Lubuntu 22.04.3 LTS (Água-viva Jammy) e, certas vezes, a impressão é interrompida por conta de número de folhas inferior do necessário para imprimir um certo documento. Tudo bem, até aí é normal.
Porém, quando selecionamos um **certo intervalo de páginas** e consequentemente clicamos para a impressão, a saída **não é a mesma da entrada** que selecionamos no **intervalo de páginas**. O Lubuntu **não reconhece nosso comando**, e **então começa toda a impressão, desde o início, fazendo gastar MUITA folha.**
Minha sugestão é que **concertem isso necessariamente**. Também coloquem algum software nativo no Lubuntu para acompanhar a impressão (A janela de propriedades de impressão do próprio sistema não salva as configurações, como: colocar tons de cinza).

Acredito que existem muitas pessoas por aí com este **problema**, então espero que lancem uma atualização.

**Comment by tsujan** [comment]
LXQt has nothing to do with Lubuntu. This is an LXQt Discussions page.
